lib/util/string_conv/string_conv.h

```diff
@@ -0,0 +1,56 @@
+/*
+ * Copyright (c) 2022 Nordic Semiconductor ASA
+ *
+ * SPDX-License-Identifier: Apache-2.0
+ */
+
+#ifndef STRING_CONV_H__
```

Comment:
use name that contains path as well. for example https://github.com/zephyrproject-rtos/zephyr/blob/c31cb0075ad1933da5487db734c118cd5083ae3d/include/sys/cbprintf.h#L7